pkg/controllers/subnetbinding/subnetbinding_controller.go

```diff
@@ -203,52 +206,105 @@ func (r *Reconciler) listBindingMapIDsFromCRs(ctx context.Context) (sets.Set[str
 	return bmIDs, nil
 }
 
+func getVpcPath(subnetPath string) (string, *errorWithRetry) {
+	info, err := servicecommon.ParseVPCResourcePath(subnetPath)
+	if err != nil {
+		return "", &errorWithRetry{
+			message: fmt.Sprintf("Invalid Subnet path %s", subnetPath),
+			retry:   false,
+			error:   fmt.Errorf("failed to parse Subnet path %s", subnetPath),
+		}
+	}
+	return fmt.Sprintf("/orgs/%s/projects/%s/vpcs/%s", info.OrgID, info.ProjectID, info.VPCID), nil
```

Comment:
VPCResourceInfo has the public function `GetVPCPath`, we can call it directly?

Reply:
Updated, thanks!